### ISSUE TITLE
update Izzy on Droid version

### ISSUE THREAD

**Issue by martianhi** [ethereal-developers/OpenScan#110]
for me the last update on Izzy on Droid is from three years ago.

Seeing as it isn't yet in FDroid itself, would it be poss to update the Izzy version?

**Comment by veejayts** [comment]
You're right, v2.2.1 didn't get a tagged github release as it was very minor. The latest version should get picked up soon enough though. Thanks for using OpenScan.
